recipes/blaspp/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "blaspp" %}
+{% set version = "2020.10.02" %}
+{% set build = 0 %}
+{% set sha256 = "36e45bb5a8793ba5d7bc7c34fc263f91f92b0946634682937041221a6bf1a150" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://bitbucket.org/icl/blaspp/downloads/blaspp-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # Fix MSVC builds
+    # https://bitbucket.org/icl/blaspp/pull-requests/31/wip-msvc-windows-builds
+    - 0001-GNU-extension-__attribute__-format.patch
+    - 0002-Complex-Support-OpenBLAS-w-MSVC.patch
+    - 0003-Missing-include-for-std-max.patch
+    - 0004-Install-RUNTIME.patch
+
+build:
+  number: {{ build }}
+
+  run_exports:
+    # no ABI compatibility between releases
+    - blaspp {{ version }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make           # [unix]
+    - ninja          # [win]
+    - cmake >=3.15.0
+    - llvm-openmp    # [osx or linux]
+  host:
+    - {{ blas_impl }}
+  run:
+    - libblas * *{{ blas_impl }}
+    - {{ blas_impl }}
```

Comment:
```suggestion
  host:
    - libblas
    - libcblas
```

Reply:
Are those OpenMP enabled?

Reply:
There are openmp enabled variants and there are ones without. Using this in host enables the use of any BLAS backend at install time by the user. (This is similar to `update-alternatives` on debian)